data/mods/No_Hope/Mapgen/mil_base_z0.json

```diff
@@ -557,7 +557,7 @@
         { "vehicle": "military_vehicles", "x": 79, "y": 42, "chance": 25, "status": 1, "rotation": 90 },
         { "vehicle": "military_vehicles", "x": 86, "y": 13, "chance": 25, "status": 1, "rotation": 270 },
         { "vehicle": "military_vehicles", "x": 79, "y": 54, "chance": 25, "status": 1, "rotation": 270 },
-        { "vehicle": "forklift", "x": 62, "y": 62, "chance": 80, "rotation": 90 }
+        { "vehicle": "forklift", "x": 62, "y": 62, "chance": 80, "status": 0, "rotation": 90 }
```

Comment:
Not needed. The default status is -1, which means it's lightly damaged. I see no reason for a forklift to be ideally undamaged with status 0.

Reply:
@Night-Pryanik
I went two ways on this. My ultimate thinking was that since the forklift is safely stored in a closed and locked warehouse _where all other items are untouched and undamaged_, it would most likely have been untouched since the cataclysm began and therefore in perfect condition.